src/components/ChrisAPOD.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const RightOfImg = styled.div`
+    padding-left: 25px;
+`;
+
+const ChrisAPOD = props => {
+    return (
+        <>
+            <img src={props.nasaData.url} />
```

Comment:
So sometimes the url returns a video, so look at the properties that you are getting inside the `response.data` see which property would tell you whether it's a video or an image. According to that property, render either an image, or a video